docs/products/kafka/get-started.md

```diff
@@ -4,19 +4,388 @@ sidebar_label: Get started
+Start using Aiven for Apache Kafka® by setting up and configuring a service, connecting to it, and managing your data.
+
+## Prerequisites
+
+Before you begin, ensure you have access to the following tools:
+
+- [Aiven Console](https://console.aiven.io)
+- [Aiven Provider for Terraform](https://registry.terraform.io/providers/aiven/aiven/latest/docs)
+- [Aiven CLI](/docs/tools/cli)
 
 ## Create an Aiven for Apache Kafka® service
 
-<CreateService serviceType="Apache Kafka®" />
+You can create your Aiven for Apache Kafka service using the Aiven Console, the Aiven CLI,
+or automate the process with Terraform.
+
+<Tabs groupId="group1">
+<TabItem value="1" label="Console" default>
+
+<CreateService serviceType="Apache Kafka®"/>
+
+</TabItem>
+<TabItem value="2" label="Terraform">
+
+1. [Create a personal token](/docs/platform/howto/create_authentication_token).
+1. Create a `sample.tf` file for the `aiven` provider configuration and the `aiven_kafka`
+   resource.
+
+   ```hcl
+   variable "aiven_token" {
+     type = string
+   }
+
+   variable "aiven_project_name" {
+     type = string
+   }
+
+   terraform {
+     required_providers {
+       aiven = {
+         source  = "aiven/aiven"
+         version = ">=4.0.0, <5.0.0"
+       }
+     }
+   }
+
+   provider "aiven" {
+     api_token = var.aiven_token
+   }
+
+   resource "aiven_kafka" "kafka" {
+     project                 = var.aiven_project_name
+     cloud_name              = "google-europe-west1"
+     plan                    = "startup-16"
+     service_name            = "my-kafka"
+     maintenance_window_dow  = "friday"
+     maintenance_window_time = "23:00:00"
+
+     kafka_user_config {
+       kafka_version = "3.7"
+     }
+   }
+
+   output "kafka_service_host" {
+     value = aiven_kafka.kafka.service_host
+   }
+
+   output "kafka_service_port" {
+     value = aiven_kafka.kafka.service_port
+   }
+
+   output "kafka_service_username" {
+     value = aiven_kafka.kafka.service_username
+   }
+
+   output "kafka_service_password" {
+     value     = aiven_kafka.kafka.service_password
+     sensitive = true
+   }
+   ```
+
+1. Create a `terraform.tfvars` file to assign values to your declared variables.
+
+    ```hcl
+    aiven_token       = "AIVEN_TOKEN"
+    aiven_project_name = "PROJECT_NAME"
+    ```
+
+1. Run the following commands to initialize and apply the configuration:
+
+   ```bash
+   terraform init
+   terraform plan
+   terraform apply --auto-approve
+   ```
+
+1. Store the Terraform outputs in environment variables to use them when [connecting](#connect):
+
+    ```bash
+    KAFKA_HOST="$(terraform output -raw kafka_service_host)"
+    KAFKA_PORT="$(terraform output -raw kafka_service_port)"
+    KAFKA_USER="$(terraform output -raw kafka_service_username)"
+    KAFKA_PASSWORD="$(terraform output -raw kafka_service_password)"
+    ```
+
+</TabItem>
+<TabItem value="3" label="CLI">
+
+Open your terminal and run the following command:
+
+```bash
```

Comment:
Could we recommend customers enable tiered storage by default here? We need to get more customers to use it